### PR TITLE
Simple Text Fix ( Resubmit )

### PR DESCRIPTION
**Pull request by DaveG9856**
AS requested Changed arriving to into "Travelling to Pokestop: {0} ({1}m Away)"),

### CODE REVIEW

File: PoGo.PokeMobBot.Logic/Common/Translations.cs [hunk 1]
```diff
@@ -191,7 +191,7 @@ public class Translation : ITranslation
             new KeyValuePair<TranslationString, string>(TranslationString.EventFortFailed,
                 "Name: {0} INFO: Looting failed, possible softban. Unban in: {1}/{2}"),
             new KeyValuePair<TranslationString, string>(TranslationString.EventFortTargeted,
-                "Arriving to Pokestop: {0} in ({1}m)"),
+                "Travelling to Pokestop: {0} ({1}m) Away"),
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Common/Translations.cs]: Should better be Travelling to Pokestop: {0} ({1}m Away)

**Comment by DaveG9856** [comment]
yep, I copied the wrong one sorry bout that. all fixed ( again ;) )

**Comment by DurtyFree** [comment]
Thx @DaveG9856 :)

**Comment by FabryB** [comment]
You forgot a final ) on the line, so the project is not even compiling... How did you test ???

**Pull request merged**